zipkin-storage/elasticsearch/src/main/resources/zipkin/storage/elasticsearch/zipkin_template.json

```diff
@@ -12,6 +12,16 @@
           "filter": [
             "lowercase"
           ]
+        },
+        "traceId_analyzer": {
```

Comment:
this might need to be reconsidered if we want to support elasticsearch 5 https://github.com/openzipkin/zipkin/issues/1379